src/current/_includes/v24.1/sql/show-ranges-output-deprecation-notice.md

```diff
@@ -1,4 +1,4 @@
-The statement syntax and output documented on this page use the updated `SHOW RANGES` that **will become the default in CockroachDB v23.2**. To enable this syntax and output, set the [cluster setting `sql.show_ranges_deprecated_behavior.enabled`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-sql-show-ranges-deprecated-behavior-enabled)  to `false`:
+The statement syntax and output documented on this page use the updated `SHOW RANGES` that **will become the default in CockroachDB v23.2**. To enable this syntax and output, set the [cluster setting `sql.show_ranges_deprecated_behavior.enabled`]({% link {{ page.version.version }}/cluster-settings.md %})  to `false`:
```

Comment:
I'm confused about the reference to 23.2 here in the 24.1 version of this file. What am I missing?